pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -71,7 +71,7 @@ func LowNodeUtilization(ctx context.Context, client clientset.Interface, strateg
 	}
 	thresholdPriority, err := utils.GetPriorityFromStrategyParams(ctx, client, strategy.Params)
 	if err != nil {
-		klog.V(1).Infof("failed to get threshold priority from strategy's params: %#v", err)
+		klog.ErrorS(err, "failed to get threshold priority from strategy's params")
```

Comment:
Let's be consistent and stick with Infos and have err="..."

Reply:
Agree, and this is debateably what's recommended by the upstream structured logging migration instructions: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#using-errors

> For expected errors (errors that can happen during routine operations) please consider using klog.InfoS and pass error in err key instead.

Depending if we see this as "routine" or not